mkdocs.yml

```diff
@@ -0,0 +1,83 @@
+# Site
```

Comment:
This file is where the magic happens. MkDocs looks at this file to determine how to generate the static website.